src/auth/oauth2client.ts

```diff
@@ -342,8 +338,7 @@ export class OAuth2Client extends AuthClient {
     if (opts.scope instanceof Array) {
       opts.scope = opts.scope.join(' ');
     }
-    const rootUrl =
-        this.authBaseUrl || OAuth2Client.GOOGLE_OAUTH2_AUTH_BASE_URL_;
+    const rootUrl = OAuth2Client.GOOGLE_OAUTH2_AUTH_BASE_URL_;
```

Comment:
Why remove `authBaseUrl`?

Reply:
It's another one of those properties that doesn't really have a reason to exist.  I can only assume it was here for unit testing purposes, but there are far better ways to do that.